ibtk/src/lagrangian/LEInteractor.cpp

```diff
@@ -4264,6 +4264,7 @@ LEInteractor::spread(double* const q_data,
     const int stencil_size = getStencilSize(spread_fcn);
     const int min_ghosts = getMinimumGhostWidth(spread_fcn);
     const int q_gcw_min = q_gcw.min();
+    TBOX_ASSERT(0 <= q_gcw_min);
```

Comment:
Since we error out if `q_gcw_min < min_ghosts` below this is redundant, but I added it anyway since it makes it clearer that this value has to be nonnegative.